shared/chat/conversation/list.desktop.js

```diff
@@ -389,37 +406,128 @@ const listStyle = {
   paddingBottom: listBottomMargin,
 }
 
-class CellSizeCache extends defaultCellMeasurerCellSizeCache {
-  _indexToID: (index: number) => any;
-
-  constructor (indexToID) {
-    super({uniformColumnWidth: true, uniformRowHeight: false})
-    this._indexToID = indexToID
+let lastMessageCount
+function chatCellRangeRenderer (messageCount: number, cellSizeCache: any, {
```

Comment:
a copy/paste fork of https://github.com/bvaughn/react-virtualized/blob/master/source/Grid/defaultCellRangeRenderer.js#L1